.github/workflows/sycl-post-commit.yml

```diff
@@ -60,21 +60,6 @@ jobs:
             image_options: -u 1001 --device=/dev/dri --device=/dev/kfd
             target_devices: ext_oneapi_hip:gpu
             reset_intel_gpu: false
-          - name: E2E tests on Intel Ponte Vecchio GPU
-            runner: '["Linux", "pvc"]'
-            env: '{"LIT_FILTER_OUT":"ESIMD/unified_memory_api/"}'
```

Comment:
should we have it precommit only and not in both? is the limiting factor the number of machines?

Reply:
I think that should be a broader question and not specific to PVC.

Reply:
I don't think the number of PVC machine is the limiting factor here. I recall some work being done to move all post-commit testing to pre-commit. If that's still our long-term plan, I think we should enable PVC on pre-commit only (and perhaps Nightly). Otherwise, I don't see any technical limitation towards enabling PVC on both pre and post commit.

Reply:
yeah i guess this kind of falls back to what should we test in precommit vs postcommit vs nightly, my initial reaction is precommit should test things/build configurations most important and most likely for users to break/care about, postcommit should be one level below, and nightly one level below that, but im fine with precommit only